.travis/opensuse.sh

```diff
@@ -7,6 +7,8 @@ ${DOCKER_EXEC} zypper -n install time.x86_64
 ${DOCKER_EXEC} zypper -n ref
 ${DOCKER_EXEC} zypper -n install rpmdevtools
 ${DOCKER_EXEC} rpmdev-setuptree
+${DOCKER_EXEC} zypper -n ar -f -G http://ftp.pbone.net/mirror/ftp5.gwdg.de/pub/opensuse/repositories/Virtualization/openSUSE_Leap_42.1/Virtualization.repo
+${DOCKER_EXEC} zypper -n ref
```

Comment:
Are we expecting open source consumers to add this repo to their zypper configuration? That could cause a great deal of confusion, especially for less experienced users. I don't think we should be adding any requirements that don't come from the official OpenSUSE repos.